packages/bot-engine/src/services/tradeEngine/utils/client.js

```diff
@@ -0,0 +1,15 @@
+export const client = (() => {
```

Comment:
Can we not get this information from the store passed to the trade engine?

Reply:
@aaron-binary since core also reading in from local-storage, i don't think there is a need to get it from there. unless i have to when i'm doing mobile which i will update it then. this probably will change when we implement mobile app.